build/test-scripts/wasm-run-automated-uitests.sh

```diff
@@ -35,10 +35,10 @@ export UNO_TESTS_RESPONSE_FILE=$BUILD_SOURCESDIRECTORY/build/nunit.response
 if [ "$UITEST_AUTOMATED_GROUP" == 'Default' ];
 then
 	export TEST_FILTERS=" \
-		Namespace != SamplesApp.UITests.Snap \
```

Comment:
Ahh, https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests.Generator/SnapShotTestGenerator.cs is not adding the Property attribute from what I see? So it makes sense that this filter won't work.

Reply:
Thanks, will investigate that change then :)